Utkarsh Somvanshi/PersonalPortfolio/styles.css

```diff
@@ -0,0 +1,223 @@
+    font-family: 'Kanit', sans-serif;
+    box-sizing: border-box;
+}
+section {
+    background-color: yellow;
+    border: 1px solid black;
+}
+.first {
+    background-color: rgb(5, 5, 44);
+    height: 100vh;
+    color: white;
+    font-size: 30px;
+}
+ul {
+    display: flex;
+}
+ul li {
+    list-style: none;
+    width: 130px;
+    text-decoration: none;
+    padding: 0px 10px 0px 10px;
+}
+ul li:hover {
+    transition: 0.5s all;
+    background-color: coral;
+    border-radius: 5px;
+    cursor: pointer;
+}
+ul li a {
+    text-decoration: none;
+    list-style: none;
+    color: white;
+}
+.logo {
+    padding-left: 20px;
+}
+nav {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 20px;
+}
+.intro {
+    margin: 50px 50px 150px 80px;
+}
+.introSection {
+    display: flex;
+    height: 100%;
+    justify-content: space-between;
+    align-items: center;
+}
+.name {
+    font-size: 100px;
+}
+.AboutMeBtn {
+    padding: 30px 80px 30px 80px;
+    margin: 50px 10px 0px 0px;
+    border-radius: 10px;
+    border: none;
+    cursor: pointer;
+    font-size: 20px;
+    background-color: /*rgb(251, 80, 80)*/ coral;
+}
+.AboutMeBtn:hover {
+    transition: 0.3s all;
+    background-color: rgb(251, 80, 80);
+}
+.AboutMeBtn:active {
+    background-color: yellow;
+}
+.cvbtn {
+    padding: 30px 80px 30px 80px;
+    border-radius: 10px;
+    border: none;
+    cursor: pointer;
+    font-size: 20px;
+    background-color: /*rgb(251, 80, 80)*/coral;
+    margin-top: 50px;
+}
+.cvbtn:hover {
+    transition: 0.5s all;
+        background-color: rgb(251, 80, 80);
+}
+.cvbtn:active {
+    background-color: yellow;
+}
+.socialmediaLogo {
+    height: 80px;
+    padding-bottom: 40px;
+    cursor: pointer;
+    display: flex;
+}
+.second {
+    height: 700px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: white;
+}
+.information {
+    margin: 80px;
+}
+.devimg {
+    position: relative;
+    right: 30px;
+}
+.third {
+    background-color: rgb(5, 5, 44);
+}
+.skills {
+    color: white;
+    margin: 80px;
+    font-size: 30px;
+}
+.skillsbar{
+    margin: 30px;
+}
+.fourth {
+    background-color: white;
+    margin: 80px;
+    border: none;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+}
+.projects {
+    width: 400px;
+    height: 500px;
+    margin: 20px;
+    box-shadow: 5px 5px 25px grey;
+}
+.projectContainer {
+    display: flex;
+    justify-content: space-around;
+}
+@media screen and (max-width: 1000px) {
+    .projectContainer {
+        display: flex;
+        flex-wrap: wrap;
+    }
+}
+.subtext {
+    margin: 20px;
+}
+.subtext a {
+    text-decoration: none;
+    list-style: none;
+    color: rgb(251, 80, 80);
+}
+.projectImage {
+    margin: 10px;
+    border-radius: 150px;
+    margin-bottom: 50px;
+}
+hr {
+    margin: 0px 20px 0px 20px;
+}
+.contacts {
+    padding: 80px;
+    background-color: rgb(5, 5, 44);
+}
+form {
```

Comment:
Please take care of the form tag as this looks a bit unattractive because it is not in the center. Also, the section doesn't have any responsive features which further adds to the concern.